recipes/mentalist/0.1.4/meta.yaml

```diff
@@ -10,20 +10,20 @@ source:
   sha256: 7dc6bd983156a28d59ad480bea1d7761ebdcd25087db73ef9ce3fb0e59cd224b
 
 build:
-  number: 1
+  number: 2
   skip: True  # [osx]
 
 requirements:
-  build:
-    - julia 0.5.2
+  host:
+    - julia
     - hdf5
-    - mpfr
-    - libxml2 2.9.*
+    - {{ pin_compatible('mpfr') }}
```

Comment:
I don't think this will work. `mpfr` should be fine in both host/run.

Reply:
Could you help me interpret this error message?

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - julia -> mpfr=3.1
  - mpfr=4
```

Reply:
it means the Julia package that was installed was linked against a older version than what we want to have today. This is most likely because you have used the bioconda Julia version, but you want to have the most recent conda-forge vesion. I have fixed this for you hopefully.

Reply:
And this error now means we need to rebuild julia in the conda-forge feedstock :(
You are maintainer, so let do this together :)

Reply:
`mentalist-0.1.x` is designed to run on `julia-0.5.x` and `mentalist-0.2.x` is designed to run on `julia-0.6.x`.

Reply:
What do you mean by 're-build'? What (if anything) needs to change in the `julia` recipe?